src/main/java/com/esri/samples/raster/raster_rendering_rule/RasterRenderingRuleSample.java

```diff
@@ -0,0 +1,166 @@
+/* Copyright 2019 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.samples.raster.raster_rendering_rule;
+
+import java.util.List;
+
+import com.esri.arcgisruntime.arcgisservices.RenderingRuleInfo;
+import com.esri.arcgisruntime.layers.RasterLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.raster.ImageServiceRaster;
+import com.esri.arcgisruntime.raster.RenderingRule;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.ComboBox;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+public class RasterRenderingRuleSample extends Application {
+
+    private ComboBox<String> renderingRuleDropDownMenu;
+    private MapView mapView;
+    private ArcGISMap map;
+    private final String imageServiceUrl = "http://sampleserver6.arcgisonline.com/arcgis/rest/services/CharlotteLAS/ImageServer";
+
+    @Override
+    public void start(Stage stage) {
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Raster Rendering Rule Sample");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // create drop down menu of Rendering Rules
+            renderingRuleDropDownMenu = new ComboBox<>();
```

Comment:
Call this `renderingRuleComboBox` or `renderingRuleInfoComboBox`. Our convention is to have the UI control name as the last part of the variable name.